extension/observer/cfgardenobserver/testdata/config.yaml

```
@@ -0,0 +1,4 @@
+cfgarden_observer:
+cfgarden_observer/all_settings:
+  refresh_interval: 20s
+  endpoint: http://localhost:8080b
```

```suggestion
  endpoint: http://localhost:8080
```
I assume `b` is a typo?